cibyl/plugins/openstack/test_collection.py

```diff
@@ -0,0 +1,56 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+from cibyl.models.model import Model
+
+# pylint: disable=no-member
+
+
+class TestCollection(Model):
+    """ Model for a collection of tests run in an Openstack deployment."""
+
+    API = {
+        'tests': {
+            'arguments': []
+        },
+        'setup': {
+            'attr_type': str,
+            'arguments': []
+        }
+    }
+
+    def __init__(self, tests: set = None, setup: str = None):
```

Comment:
Use Optional[...] for types that are nullable. The type hint here says that these should not be None.

I am also missing the type for 'tests' here, what is supposed to be stored in them?

Reply:
These are sets of strings, updated the type hint with the Optional, thanks for the suggestion